docs/debugging.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[debugging]]
+== Debugging the agent itself
+
+Hopefully the agent Just Works™ for you but depending on your situation the agent could need some tuning.
+
+Firstly, to know more about what's going on inside the agent, you can increase the amount of log messages it writes by setting the log level with the option `log_level = 0` -- `0` being the level of most messages, `DEBUG`.
```

Comment:
Not directly related to docs, but do you think it would be be helpful to enable users to use log level names, like "DEBUG" or "INFO"? What's typical in Ruby apps?

Reply:
It's common to use the constants `Logger::DEBUG` (`== 0`) for example. I just thought this was easier than explaining how to get to those depending on your situation. Via ENV you cant, in the config file you need the template tags `<%= Logger::DEBUG %>`, in normal ruby they're just there. The Getting Started docs assume you configure with a config file, so I could just assume the same here and let people figure out themselves how it might be different in their situation?

Reply:
To your question: Could be a _nice to have_ to convert `:debug` to `Logger::DEBUG` I suppose. But not common or expected, in my experience.